### PR TITLE
Remove css referencing personas

### LINKED ISSUE #12130: Remove css personas referencing non-existing files
During the `compress_assets` task:

```
 - Could not find file /code/src/olympia/../../static/css/zamboni/../../img/personas/slideshow-nav.png
 - Could not find file /code/src/olympia/../../static/css/zamboni/../../img/zamboni/personas/theme-promo.png
 - Could not find file /code/src/olympia/../../static/css/zamboni/../../img/zamboni/personas/complete-theme-promo.png
 - Could not find file /code/src/olympia/../../static/css/zamboni/../../img/zamboni/personas/submit.jpg
 - Error finding 4 images
```

### PR DESCRIPTION
**Pull request by wagnerand**
Fix #12130.

### CODE REVIEW

File: static/css/zamboni/zamboni.css [hunk 1]
```diff
@@ -1256,88 +1256,6 @@ ul.license li.copyr { background-position: 0 -260px; }
     color: #390;
 }
```
Comment (eviljeff) [inline static/css/zamboni/zamboni.css]: I'd think _any_ rule referencing `.persona` is dead wood now.  (not `.persona-license` though, I unwisely re-used it for the static theme license chooser)

File: static/css/zamboni/themes.less [hunk 2]
```diff
@@ -47,12 +47,6 @@ aside.profile {
 
         + p {
             margin-bottom: 15px;
-
```
Comment (eviljeff) [inline static/css/zamboni/themes.less]: I think this entire file can go. Theme submission is in devhub which has it's own styles.

**Comment by eviljeff** [comment]
As long as travis agrees, obviously

File: static/css/zamboni/zamboni.css (hunk 1 above)
Comment (wagnerand) [reply static/css/zamboni/zamboni.css]: D'oh!

**Comment by wagnerand** [comment]
Removing some disco-pane cruft along the way.

**Pull request merged**